packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -116,6 +116,8 @@ export function useDrag(options: DragOptions): DragResult {
     }
 
     let items = options.getItems();
+    // Clear existing data (e.g. selected text on the page would be included in some browsers)
+    e.dataTransfer.clearData();
```

Comment:
```suggestion
    e.dataTransfer.clearData?.();
```
wonder if it is worth doing this instead of the mock so people don't have their tests break on them